docs/advanced/app-connectors.md

```diff
@@ -44,7 +47,7 @@ export default createAppConnector(() => ({
 
 From the point of view of a consumer of your connector, the resulting object will be a function that can get used to connect an app.
 
-For example, this is how app authors can initiate a connection to the Voting app. In this example, `connectVoting` got returned by `createAppConnector()`:
+For example, this is how app authors can initiate a connection to the Voting app. In this example, `connectVoting` got returned by the implementation of `createAppConnector()` of the Voting connector:
```

Comment:
I’m not sure I understand this, the implementation of `createAppConnector()` is not in the `Voting` connector? :thinking:

Reply:
You are right, I tried to clarify and I complicate it more 😅